packages/middleware/src/helpers/metrics.ts

```diff
@@ -0,0 +1,25 @@
+import { RequestHandler } from "express";
+import prometheusClient from "prom-client";
+
+const prometheusRegister = new prometheusClient.Registry();
+prometheusRegister.setDefaultLabels({
+  app: "middleware",
+});
+prometheusClient.collectDefaultMetrics({ register: prometheusRegister });
```

Comment:
I think default metrics are metrics that are largely already available in console (CPU/Memory) no?

Reply:
What would be useful is to keep average response time per endpoint

Reply:
GC and heap could be useful I suppose, but yeah average response time per endpoint would definitely be useful as well. The page views initiative will also rely on prometheus